spec/test_app.rb

```diff
@@ -33,4 +35,23 @@ class TestApp < Sinatra::Base
       </div>
     </div>"
   end
+
+  get '/options' do
+    option_1 + option_2
```

Comment:
I'd just put the HTML directly in this method, I don't see the point of the variables if they are not re-used. Makes it harder to read actually.